scripts/Engineering/EnggUtils.py

```diff
@@ -393,12 +395,18 @@ def write_ENGINX_GSAS_iparam_file(output_file, difa, difc, tzero, bank_names=Non
         template_file = 'template_ENGINX_241391_236516_North_and_South_banks.prm'
     import os
     template_file = os.path.join(os.path.dirname(__file__), template_file)
+
     if not bank_names:
         bank_names = ["North", "South"]
 
     with open(template_file) as tf:
         output_lines = tf.readlines()
 
+    gamma_0_gamma_1 = []
```

Comment:
Sorry I might have changed my mind on this one, but I don't think this is strictly valid as we're sort of merging parameters refinend for different peak profiles (though I doubt it would actually cause any problems if we kept it). 
As I see it we have we have three options (in order of preference): 
(1) I forget if GSAS has two separately numbered profiles for the back to back exp convoluted with a Gaussian (mantid function `BackToBackExponential`) and a pseudo-voigt (mantid function `Bk2BkExpConvPV`). It might not, as obviously you can just set the lorz gammas of the PV function to zero. If it does have separate profiles and it is possible to specify parameters for two different peak profiles in the same .prm, then we can keep the peak parameters from the template .prm for the PV but add lines for the Gaussian conv profile. Long term we can also store parameters for the `Bk2BkExpConvPV` in the Parameters.xml as well.
(2) Keep the profile number as is and just set Gammas to 0 (see above)
(3) Leave them as is as these parameters - this means the peak will be slightly different but it probably won't be that significant (mostly effect the width) and they will probably be refined for each sample anyway.
What do you think?

Reply:
This is interesting. I am tempted to say number 2 so set gamma values to 0. 

As for number1: trying to figure out whether the values are separate or not. Do you have an idea of how we could check this? The profile I've been basing the above changes on (TOF profile function 3) on the GSAS pdf mentions that:
` "It is a convolution of the back-to-back exponentials used in the first function with the pseudo-Voight used in the second function."`
I think this is suggesting that the profiles are not separate. It seems than TOF profile function 1 is convolution of 2 bk2bkexponentials with Gaussian and ToF profile 2 is a Pseudo-Voigt.

I don't know if it's possible to define different sets of peak profile parameters in the PRM file. From some googling of GSAS and PRCF it doesn't seem to be the done thing to define different peak profile but I'm wondering if this PRCF1 line defines the profile used (3 in our case, but we could separately have values for profile 1?)

Reply:
> I think this is suggesting that the profiles are not separate. It seems than TOF profile function 1 is convolution of 2 bk2bkexponentials with Gaussian and ToF profile 2 is a Pseudo-Voigt.

This is what I meant by separate profiles, so we could instead put parameters for profile 1 (just gaussian componeont), rather than profile 3

> I don't know if it's possible to define different sets of peak profile parameters in the PRM file. From some googling of GSAS and PRCF it doesn't seem to be the done thing to define different peak profile but I'm wondering if this PRCF1 line defines the profile used (3 in our case, but we could separately have values for profile 1?)

 I think the line that sets the peak profile to 3 is the line (but I don't know what the other parameters are)
`1PRCF1     3   21   0.00050 `
If we can provide parameters to both (my option 1) then that'd be great, it might just be a case of adding a block corresponding to 
`2PRCF    1    x     x`
in analogy with how the two banks are denoted in the `_all.prm`

Reply:
This link might be useful (it also links to a .prm for POWGEN)
https://subversion.xray.aps.anl.gov/pyGSAS/Tutorials/TOF%20Calibration/Calibration%20of%20a%20TOF%20powder%20diffractometer.htm

Reply:
As we discussed, I have kept the TOF profile function number 3 with 21 parameters and set the gamma values to 0.